docs/user/datastores/query_interceptor.rst

```diff
@@ -142,5 +147,59 @@ hours, or minutes. For example:
       }
     }
 
+Percentages can be defined in decimal form. e.g. .1 corresponds to 10%. Any query smaller than the first size will
+return 100% of the records. For example:
+
+.. code-block:: none
+
+    geomesa {
+      guard {
+        graduated {
+          "sftName" = [
+            { size = 1,  percentage = .8 }
+            { size = 10, percentage = .5  }
+            { percentage = .1 }
+          ]
+        }
+      }
+    }
+
+It is also possible to specify the attribute to use for the threading key in subsampling:
+
+.. code-block:: none
+
+    geomesa {
+      guard {
+        graduated {
+          "sftName" = [
+            { size = 1,  percentage = .8, attribute = "name" }
+            { size = 10, percentage = .5, attribute = "name"  }
+            { percentage = .1, attribute = "name" }
```

Comment:
maybe we could change these name to `sampling-percentage` and `sampling-attribute` or something similar so it's more descriptive? or alternatively have a nested block like `sampling: { percentage: .1, attribute: name}`